src/components/CompareCities/CompareCities.scss

```diff
@@ -0,0 +1,61 @@
+@import '~style/variables';
+
+.wrapper {
+  @include shadow;
+
+  border-radius: 8px;
+  background: $color-white;
+  border: 1px solid $color-grey--200;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  border-bottom: 1px solid $color-grey--200;
+
+  @include for-tablet-portrait-up {
+    flex-direction: row;
+    align-items: center;
+    padding: 24px 30px;
+  }
+}
+
+.title {
+  @include type-indicator-card-title--large;
+
+  margin-bottom: 8px;
```

Comment:
Title and subtitle look slightly misaligned to me. Subtitle a little too high on the page.

Reply:
Fixed (`align-items: baseline;`)